src/Sarif.Viewer.VisualStudio.Core/Models/AnalysisStep.cs

```diff
@@ -285,6 +285,8 @@ public DelegateCommand<TreeView> SelectPreviousCommand
                         var control = treeView as TreeView;
                         var model = control.DataContext as AnalysisStep;
                         model.SelectedItem = this.FindPrevious();
+                        model.SelectedItem.NavigateTo(usePreviewPane: false, moveFocusToCaretLocation: false);
+                        control.Focus();
```

Comment:
>control.Focus() [](http://example.com/codeflow?start=24&length=15)

Keyboard shortcut only works when TreeView has focus.
Set focus back to TreeView so user can continue navigating using shortcut.